apidoc/Titanium/UI/WebView.yml

```diff
@@ -635,6 +635,10 @@ properties:
     description: |
         On the iOS platform, this is not per-webview. Once you have set this property for a webview
         it will not change for same. But while creating new webview it can be changed to new user agent.
+
+        On Android, changing the user agent after the webview has begun loading content may cause
+        the webview to reload and fire multiple `load` or `beforeLoad` events. Developers should provide the
```

Comment:
It is [beforeload](https://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.WebView-event-beforeload).

Reply:
Thanks for the catch Hans!